templates/satspay/display.html

```diff
@@ -367,6 +247,17 @@
       },
       amountDue() {
         return this.charge.amount - this.charge.balance
+      },
+      unifiedQR() {
+        const bitcoin = (this.charge.onchainaddress || '').toUpperCase()
+        let queryString = `bitcoin:${bitcoin}?amount=${
+          this.charge.amount / 1e8
```

Comment:
- JS floating point can be tricky:
![image](https://github.com/lnbits/satspay/assets/2951406/7450d42f-859e-4c16-a189-7b3837fa7272)

Reply:
better to explicitly do `this.charge.amount / 100000000` ?

Reply:
- it would give the same exponential result
- suggestion: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
This is not resolved, please have a look t [toFixed]( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) function

Reply:
ahh, i see the difference! pushed, `( this.charge.amount / 1e8).toFixed(8)`